app/models/concerns/smooch_messages.rb

```diff
@@ -286,7 +286,7 @@ def handle_bundle_messages(type, list, last, app_id, annotated, send_message = t
       if ['timeout_requests', 'menu_options_requests', 'resource_requests', 'relevant_search_result_requests', 'timeout_search_requests'].include?(type)
         key = "smooch:banned:#{bundle['authorId']}"
         if Rails.cache.read(key).nil?
-          [annotated].flatten.uniq.each { |a| self.save_message_later(bundle, app_id, type, a) }
+          [annotated].flatten.uniq.each_with_index { |a, i| self.save_message_later(bundle, app_id, type, a, i * 10) }
```

Comment:
is this just to stagger the rate at which these messages are emitted?

Reply:
Yes, like, 10 seconds between them.